### DISCUSSION TITLE
Newer API versions

### DISCUSSION THREAD

**Discussion by epenabella** [arlyon/async-stripe#507] [General]
Is there decent compatibility with the newer Stripe Versions?

The README as of today says 2020-08-27 is latest stripe version suported..

I'm rewriting a whole API to rust and this could fwark the whole thing! haha

newer versions include:
[2023-10-16] Latest
[2023-08-16]
[2022-08-01] Default

**Comment by arlyon** [comment]
Yeah, I really need to update that. The answer is we target the latest stripe API version at all times. The latest version is always here: 

https://github.com/arlyon/async-stripe/blob/master/src/resources/generated/version.rs

**Reply by arlyon** [reply]
Updated the readme to this affect! https://github.com/arlyon/async-stripe/blob/master/README.md
